src/libs/actions/Policy/Policy.ts

```diff
@@ -2241,8 +2241,8 @@ function createWorkspaceFromIOUPayment(iouReport: Report | EmptyObject): string
                 },
             },
         },
-        ...employeeWorkspaceChat.onyxSuccessData,
     ];
+    successData.push(...employeeWorkspaceChat.onyxOptimisticData);
```

Comment:
Thats a weird change and seems a bit unrelated to me. Was it intended?

Reply:
For some reason without this change I'm getting the following error: 

`Expression produces a union type that is too complex to represent.`

it also happened in the GH pipeline

Reply:
Huh thats weird. Wonder if same thing happens on main

Reply:
Yeah unfortunately we have been having TS errors with `OnyxUpdate`, for now the workaround is to apply the fix the way you did.